helm/cosmo/charts/controlplane/templates/deactivate-organization.yaml

```diff
@@ -0,0 +1,142 @@
+{{ if .Values.jobs.deactivateOrganization.enabled }}
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: "{{ include "controlplane.fullname" . }}-deactivate-organization-{{ .Values.jobs.deactivateOrganization.id }}}-{{ now | unixEpoch }}"
```

Comment:
We might run into limits here https://kubernetes.io/docs/concepts/workloads/controllers/cron-jobs/. You can remove the time component.

Reply:
Done